src/borg/testsuite/helpers.py

```diff
@@ -212,6 +213,7 @@ def test_canonical_path(self, monkeypatch):
             assert Location(location).canonical_path() == \
                 Location(Location(location).canonical_path()).canonical_path(), "failed: %s" % location
 
+    @pytest.mark.skipif(is_win32, reason='Used date format (%s) not supported on windows.')
```

Comment:
%s is not supported on windows and not supported in python in general. This currently only works because python passes the string directly to the platform dependent `strftime` function.

See also: https://stackoverflow.com/questions/11743019/convert-python-datetime-to-epoch-with-strftime

Reply:
ok, can you make a separate PR for this against master (and, if applicable, against 1.1-maint)?

Reply:
What should the PR contain? Remove this test? Or change the timestamp to a different format?

Reply:
everything that is needed to remove the problem and related workarounds/checks from master.

after that is merged, you can rebase your stuff on master to have it gone from this PR also.

Reply:
note: the problematic %s format usage is not present in current master branch any more.